services/121-service/src/payments/fsp-integration/intersolve-visa/intersolve-visa.service.ts

```diff
@@ -3,6 +3,8 @@ import { MessageContentType } from '@121-service/src/notifications/enum/message-
 import { ProgramNotificationEnum } from '@121-service/src/notifications/enum/program-notification.enum';
 import { MessageProcessTypeExtension } from '@121-service/src/notifications/message-job.dto';
 import { QueueMessageService } from '@121-service/src/notifications/queue-message/queue-message.service';
+import { PaPaymentDataDto } from '@121-service/src/payments/dto/pa-payment-data.dto';
```

Comment:
The IntersolveVisaService should not import anything from the Payments Module, since the IntersolveVisa Module does not depend on this Module.